packages/dso-toolkit/src/components/paragraph/paragraph.args.ts

```diff
@@ -0,0 +1,29 @@
+import { ArgTypes } from "@storybook/types";
+import { noControl } from "../../storybook";
+
+import { Paragraph } from "./paragraph.models.js";
+
+export interface ParagraphArgs {
+  className?: string;
```

Comment:
Het component Paragraph heeft geen optioneel classname.

Het component Paragraph heeft een variant "Disclaimer".

Dat dit vervolgens wordt geimplementeerd op het paragraph _element_ is ene implementatedetail.

Reply:
gedaan